earthfile2llb/converter.go

```diff
@@ -442,13 +442,59 @@ func (c *Converter) CopyClassical(ctx context.Context, srcs []string, dest strin
 		return err
 	}
 
+	var buildContext string
+	for _, src := range srcs {
+		// local sources that copy entire parent directores like ".." or "../" are not allowed
```

Comment:
What would happen if a user is working under `/home/user/some/path/Earthfile` and does a `COPY ../../some`, would that still hit this recursion issue?

Reply:
I don't think so because we force `--use-copy-include-patterns` when copying from parent dirs -- but let me add a test case for this to be sure